src/defaultContractAddresses.ts

```diff
@@ -0,0 +1,18 @@
+export default {
```

Comment:
@rdinicut  
What do you think about providing default addresses?

Maybe we could always require to pass the addresses to Tinlake.js or the alternative would be to make different releases of Tinlake.js

Let say: `Tinlake.js Maker Mainnet` release. Which already has the right addresses pre-configured.

@philipstanislaus What do you think?

Reply:
Considering that we want to have multiple deployments of Tinlake we should provide them in the constructor or something. The abis can contain some default addresses and maybe we use those as defaults.In that case we need to use deployment abis not test abis

Reply:
Yes thats the current idea.

Reply:
There is already an option to pass in addresses and abi. I will add a rollup on Monday to make import of defaults easier, and then a user can overwrite both the abi and the addresses when instanciating tinlake.